### ISSUE TITLE
Cache DNS requests

### ISSUE THREAD

**Issue by spotter226** [AdguardTeam/AdGuardDNS#802]
### Proposed solution

It would be great if you could add in the private DNS settings the functionality of caching DNS requests. This would save some requests, as NextDNS does.

**Comment by Batman2814** [comment]
Would love to have this feature

**Comment by spotter226** [comment]
> Da quanto ho capito, Next DNS utilizza un'impostazione TTL per farlo. È già implementato in AdGuard DNS nella pagina delle impostazioni del server

Why is it that with NextDNS at the end of the month I come up with 150,000 queries while with AdGuard I come close to 300,000?

**Comment by Batman2814** [comment]
Only for Blocked Requests not unblocked requests

**Comment by spotter226** [comment]
> Only for Blocked Requests not unblocked requests

You are wrong, look at this: https://help.nextdns.io/t/x2hp2lk/which-query-types-count-towards-the-free-300k-quota

**Comment by Batman2814** [comment]
For Adguard not Next DNS

**Comment by crafty991** [comment]
Is there any news on this? It's been almost a year.
